src/Induction/WellFounded.agda

```diff
@@ -96,23 +96,22 @@ module All {_<_ : Rel A r} (wf : WellFounded _<_) ℓ where
 module FixPoint
   {_<_ : Rel A r} (wf : WellFounded _<_)
   (P : Pred A ℓ) (f : WfRec _<_ P ⊆′ P)
-  (f-ext : (x : A) {IH IH′ : WfRec _<_ P x} → (∀ {y} y<x → IH y y<x ≡ IH′ y y<x) → f x IH ≡ f x IH′)
+  (f-ext : (x : A) {IH IH′ : WfRec _<_ P x} → (∀ {y} y<x → IH {y} y<x ≡ IH′ y<x) → f x IH ≡ f x IH′)
```

Comment:
Instance 3/8: the typechecker needs a nudge here in the type of `f-ext`, but that is all.